data-access/basic_data_access.ipynb

```diff
@@ -167,538 +248,1117 @@
+    "    .institution_id(\"UCLA\")\n",
+    "    .experiment_id([\"historical\", \"ssp370\"]) # Retrieve historical and future data \n",
+    "    .table_id(\"mon\")\n",
+    "    .grid_label(\"d01\")\n",
+    "    .variable(\"prec\") # Precipitation \n",
+    "    .processes({\n",
+    "        \"concat\": \"time\"  # Concatenate along simulation dimension\n",
+    "    })\n",
+    "    .get()\n",
+    ")\n",
+    "concat_by_time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d90a447-6fe7-4331-a3da-f0839d1a419e",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# See additional details about the function arguments by printing the docstring\n",
-    "print(get_data.__doc__)"
+    "# Example behavior of concatenating by `sim`\n",
+    "cd.reset()\n",
+    "concat_by_sim = (cd\n",
+    "    .catalog(\"cadcat\")\n",
+    "    .activity_id(\"WRF\")\n",
+    "    .institution_id(\"UCLA\")\n",
+    "    .experiment_id([\"historical\", \"ssp370\"]) # Retrieve historical and future data \n",
+    "    .table_id(\"mon\")\n",
+    "    .grid_label(\"d01\")\n",
+    "    .variable(\"prec\") # Precipitation \n",
+    "    .processes({\n",
+    "        # Concatenate along simulation dimension. You'll notice that the `sim` dim\n",
+    "        # now has `historical` and `ssp370` simulations stacked ontop of each other\n",
+    "        \"concat\": \"sim\" \n",
+    "    })\n",
+    "    .get()\n",
+    ")\n",
+    "concat_by_sim"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "351a15bd-efb5-4376-9c90-8b58fe6cfbde",
+   "id": "b91e25f1",
    "metadata": {},
    "source": [
-    "### Example 1: Time-based approach\n",
-    "Retrieve gridded data using a time-based approach. ```approach``` is an optional function argument, but the default is to use a time-based approach, so you don't actually need to set this argument. "
+    "### Processor Example 2: Filter Unadjusted Models (only for WRF data)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "fdc1b79a-aede-4f8a-8162-492c6f3dfb8f",
+   "id": "528379db",
    "metadata": {},
    "source": [
-    "#### Example 1a\n",
-    "First, let's retrieve 3 kilometer resolution statistically downscaled historical data at a monthly timestep. "
+    "By default, we filter out unadjusted WRF models. If you're interested in learning more about why, you can check out the section in our guidance where we talk about that <a href=\"https://analytics.cal-adapt.org/guidance/using_in_decision_making/#the-analytics-engine-hosts-a-priori-bias-corrected-as-well-as-non-bias-corrected-wrf-data---which-one-should-a-user-choose?\" style=\"color: blue;\" target=\"_blank\">here.</a> However, if you'd like to see all models, you can pass in `filter_unadjusted_models: False` into the processors. **However, we highly recommend ONLY using the adjusted WRF models for analysis, since the unadjusted models have biases that have not been corrected.**"
```

Comment:
It doesn't look like we explain what "unadjusted" means anywhere with regard to WRF. Add something like "these are simulations that do not have bias correction"?